test/bias-2015/test_pathogenic_classifiers.py

```diff
@@ -288,3 +287,296 @@ def test_get_pm1_very_strong():
     expected_score = 3
     expected_pm1 = "PM1 (3): Variant is in domain, P21912 where (0.970909091 %) variants are associated with pathogenicity (var score 75)."
     assert (score, pm1) == (expected_score, expected_pm1)
+
+def test_get_pm2_absent():
+    """
+    Test the get_pm2 function when the variant is absent 
+    """
+    # Initialize the variant data
+    variant = extract_from_nirvana_json.VariantData(
+        chromosome="chr1",
+        position=17350500,
+        ref_allele="A",
+        alt_allele="T",
+        variant_type="SNV",
+    )
+    
+    variant.gnomad = None
+    variant.oneKg = None
+
+    # Perform the test
+    score, pm2 = pathogenic_classifiers.get_pm2(variant)
+    expected_score = 1
+    expected_pm2 = "PM2 (1): Variant is missing from GNOMAD and 1000 Genomes"
+    assert (score, pm2) == (expected_score, expected_pm2)
+
+
+def test_get_pm2_low_af():
+    """
+    Test the get_pm2 function when the variant allele frequency is low
+    """
+    # Initialize the variant data
+    variant = extract_from_nirvana_json.VariantData(
+        chromosome="chr1",
+        position=17350500,
+        ref_allele="A",
+        alt_allele="T",
+        variant_type="SNV",
+    )
+
+    # Set gnomad and oneKg attributes
+    variant.gnomad = {"allAf": 0.005}
```

Comment:
I think the gnomad AF gets represented through a different data structure. I think thats why you're only seeing 1kg in the return string and not gnomad + 1kg

Reply:
How should I go about fixing this then? Which data structure should I use?

Reply:
I just took another look at this, its actually an issue in the code itself. I've made a new issue and assigned it to you,

https://github.com/bitscopic/BIAS-2015/issues/15

Which you can take a look at after we get the benign classifier unit tests merged in.